generated/nise/__init__.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# This file was generated
+
+from nise.enums import *          # noqa: F403,F401,H303
+from nise.errors import DriverWarning   # noqa: F401
+from nise.errors import Error     # noqa: F401
+from nise.session import Session  # noqa: F401
+
+
+def get_diagnostic_information():
```

Comment:
Does this return sensical results for NISE?

Reply:
I didn't know this function existed.

It returns:

"{'os': {'name': 'Windows', 'version': '6.1.7601', 'bits': '64'}, 'python': {'version': '3.6.6 (v3.6.
6:4cf1f54eb7, Jun 27 2018, 03:37:03) [MSC v.1900 64 bit (AMD64)]', 'bits': '64', 'is_venv': False, 'packages': [{'name': 'wxpython', 'version': '4.0.3'}, {'name': 'virtualenv', 'version': '16.0.0'}, {'name': 'tox', 'version': '3.1.2'}, {'name': 'six', 'version': '1.11.0'}, {'name': 'setuptools', 'version': '39.0.1'}, {'name': 'pytz', 'version': '2018.5'}, {'name': 'python-dateutil', 'version': '2.7.3'}, {'name': 'pytest', 'version': '3.6.3'}, {'name': 'pypubsub', 'version': '4.0.0'}, {'name': 'pyparsing', 'version': '2.2.0'}, {'name': 'pyflakes', 'version': '1.6.0'}, {'name': 'pycodestyle', 'version': '2.3.1'}, {'name': 'py', 'version': '1.5.4'}, {'name': 'pluggy', 'version': '0.6.0'}, {'name': 'pip', 'version': '18.0'}, {'name': 'packaging', 'version': '17.1'}, {'name': 'numpy', 'version': '1.15.0'}, {'name': 'niswitch', 'version': '1.0.0'}, {'name': 'nise', 'version': '1.0.1.dev0'}, {'name': 'niscope', 'version': '1.0.1.dev0'}, {'name': 'nimodinst', 'version': '1.0.0'}, {'name': 'nidmm', 'version': '1.0.1.dev0'}, {'name': 'nidcpower', 'version': '1.0.1.dev0'}, {'name': 'more-itertools', 'version': '4.2.0'}, {'name': 'mccabe', 'version': '0.6.1'}, {'name': 'matplotlib', 'version': '2.2.3'}, {'name': 'kiwisolver', 'version': '1.0.1'}, {'name': 'flake8', 'version': '3.5.0'}, {'name': 'cycler', 'version': '0.10.0'}, {'name': 'colorama', 'version': '0.3.9'}, {'name': 'attrs', 'version': '18.1.0'}, {'name': 'atomicwrites', 'version': '1.1.5'}]}, 'driver': {'name': 'NI Switch Executive', 'version': 'Unknown'}, 'module': {'name': 'nise', 'version': '1.0.1.dev0'}}
"

Reply:
Fixed